src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -939,10 +938,6 @@ private function addAssetMapperSection(ArrayNodeDefinition $rootNode, callable $
                             ->info('The directory to store JavaScript vendors.')
                             ->defaultValue('%kernel.project_dir%/assets/vendor')
                         ->end()
-                        ->scalarNode('provider')
```

Comment:
We should make it a deprecated no-op instead. FrameworkBundle is subject to the BC policy.